layers/+distribution/spacemacs-base/funcs.el

```diff
@@ -689,6 +689,7 @@ the right."
 (spacemacs|create-align-repeat-x "bar" "|")
 (spacemacs|create-align-repeat-x "left-paren" "(")
 (spacemacs|create-align-repeat-x "right-paren" ")" t)
+(spacemacs|create-align-repeat-x "backslash" "\\\\")
```

Comment:
I was getting a regexp parse error when I tried writing this as `"\\"`. Alternatively this works as `"[\\]"` if that style would be preferred.